app/views/candidate_interface/application_form/show.html.erb

```diff
@@ -110,5 +110,13 @@
 
   <div class="govuk-grid-column-one-third">
     <%= render partial: '/candidate_interface/shared/need_help', locals: { support_reference: @application_form.support_reference } %>
+
+    <% if FeatureFlag.active?('apply_again') %>
```

Comment:
Should there be a check to see if `@previous_application.present?` here?

It crashes when I run it on a non-Apply Again application and in any case the heading is redundant if we don't have anything to link to.

Reply:
good shout!

Reply:
Good catch. Thanks!